docusaurus/docs/Android/01-basics/03-getting-started.mdx

```diff
@@ -38,22 +38,27 @@ class MainActivity : AppCompatActivity() {
 }
 ```
 
-### Creating a ChatDomain
+### Adding an Offline Plugin
```

Comment:
I would say `the` might be a bit better here, so that users don't get the idea there can be multiple plugins